### PR TITLE
Fixes the mention logger.

### PR DESCRIPTION
**Pull request by CyberiumShadow**

### CODE REVIEW

**Comment by CyberiumShadow** [comment]
I may replace this with my webhook mention logger instead.

**Comment by kyranet** [comment]
image

**Comment by kyranet** [comment]
*Update didyoumean for Komada 0.20.x* Do that now, or are you going to make another PR for that?

**Comment by CyberiumShadow** [comment]
SHould be good to merge

File: monitors/nomention.js [hunk 1]
```diff
@@ -1,12 +1,9 @@
-/*
-  Piece not compatible with Komada v0.20.x
-*/
 exports.run = (client, msg) => {
   if (!msg.guild || msg.author.id === client.user.id) return;
   const user = `${msg.author.username}#${msg.author.discriminator} (${msg.author.id})`;
   const channel = `#${msg.channel.name} (${msg.channel.id}) from ${msg.guild.name}`;
-  if (msg.mentions.everyone) client.funcs.log(`${user} mentioned everyone in ${channel}`);
-  else if (msg.mentions.users.has(client.user.id)) client.funcs.log(`${user} mentioned you in ${channel}`);
+  if (msg.mentions.everyone) client.console.log(`${user} mentioned everyone in ${channel}`);
```
Comment (kyranet) [inline monitors/nomention.js]: Should use `client.emit('log', cb);` instead.

**Review by kyranet — DISMISSED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**